e2e/test/scenarios/dashboard-filters/parameters.cy.spec.js

```diff
@@ -246,14 +246,15 @@ describe("scenarios > dashboard > parameters", () => {
     cy.findByText("Remove").click();
     cy.location("search").should("eq", `?${endsWith.slug}=zmo`);
 
-    cy.button("Save").click();
+    saveDashboard();
+
+    cy.log(
+      "There should only be one filter remaining and its value is preserved",
+    );
 
-    cy.log("There should only be one filter remaining and its value cleared");
     filterWidget().contains(new RegExp(`${endsWith.name}`, "i"));
-    cy.location("search").should("eq", `?${endsWith.slug}=`);
 
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("37.65");
+    cy.location("search").should("eq", `?${endsWith.slug}=zmo`);
```

Comment:
idea of this test was in checking the ability to remove parameter, after removal other filters should keep working. 

as we changed the default behaviour for filters state, URL is not updated and it doesn't make sense to verify what's rendered as a new query is not fired